test/application/certificate/verify-v3.test.js

```diff
@@ -11,7 +11,7 @@ describe('Certificate test suite', function () {
         let certificate;
 
         beforeEach(async function () {
-          sinon.stub(ethereumExplorer, 'getEtherScanFetcher').resolves({
+          sinon.stub(Explorers, 'getBitcoinTransactionFromApi').resolves({
```

Comment:
this was Ethereum before, why the change to the bitcoin one?

Reply:
That's weird, I don't remember changing that file at all. I'll update.